src/scripts/scroll_to_bottom.js

```diff
@@ -1,27 +1,36 @@
 import { keyToClasses, keyToCss } from '../util/css_map.js';
 import { translate } from '../util/language_data.js';
 import { pageModifications } from '../util/mutations.js';
-import { buildStyle } from '../util/interface.js';
+import { blogViewSelector, buildStyle } from '../util/interface.js';
 
 const scrollToBottomButtonId = 'xkit-scroll-to-bottom-button';
 $(`[id="${scrollToBottomButtonId}"]`).remove();
+const activeClass = 'xkit-scroll-to-bottom-active';
 
-const knightRiderLoaderSelector = `main ${keyToCss('loader')} ${keyToCss('knightRiderLoader')}`;
+const loaderSelector = `
+${keyToCss('timeline', 'blogRows')} > ${keyToCss('loader')},
+${keyToCss('notifications')} + ${keyToCss('loader')}
+`;
 
 let scrollToBottomButton;
-let scrollToBottomIcon;
 let active = false;
 
 const styleElement = buildStyle(`
 ${keyToCss('isPeeprShowing')} #${scrollToBottomButtonId} {
-  opacity: 0;
-  pointer-events: none;
+  display: none;
```

Comment:
The hiding CSS was the way it was for a reason; it allows the scroll to bottom button to fade out/in with the scroll to top button when peepr is opened/closed.
```suggestion
  opacity: 0;
  pointer-events: none;
```

Reply:
Hmmm. Makes sense; I guess non-modal blog view will need a different solution.

Reply:
Almost works, but it glitches briefly when you scroll up in non modal blog view. Oh well, really not important.

```diff
  ${keyToCss('isPeeprShowing')} #${scrollToBottomButtonId} {
    opacity: 0;
    pointer-events: none;
  }

+ ${keyToCss('isPeeprShowing')} button:not(${keyToCss('hidden')}) + #${scrollToBottomButtonId} {
+   display: none;
+ }
```